rhacs: fix s3 compat endpoint docs (#78980)
openshift/openshift-docs: modules/s3-api-compatible-services-configuring-acs.adoc

. For *Schedule*, select the backup frequency as daily or weekly, and select the time to run the backup process.
. Enter the *Bucket* name where you want to store the backup.
. Optionally, enter an *Object Prefix* if you want to save the backups in a specific folder structure.

> . Enter the *Endpoint* under which the S3 compatible service is reachable. Defaults to https if the scheme is left out.

Reviewer: I'm kind of unsure of what this means - does it mean that RHACS attempts to use https for the connection to the endpoint if no connection type is specified? (I'm not sure what "scheme" means in this context.)

Author: Yes, in `https://play.min.io` the scheme is `https`. It means that if you enter `play.min.io`, the backend will change this to `https://play.min.io`.

> . Enter the *Endpoint* under which the S3 compatible service is reachable. Defaults to https if the scheme is left out.

Reviewer: ```suggestion
. Enter the *Endpoint* under which the S3 compatible service is reachable. If no scheme is specified, the default, `https`, is used.
```
Would this be accurate?

Author: Yes 👍
